solutions/03-optimizing-batch-writes-updates/kwikshoppr/java/src/test/java/com/cockroachlabs/university/batch/TestArrayBindingBulkUpdates.java

```diff
@@ -58,40 +59,85 @@ public void testUpdateOrders_WithSingleBatch(int batchSize){
     }
 
     @ParameterizedTest
-    @ValueSource(ints = { 1000, 2000, 10000})
-    public void testUpdateOrders_ArrayBindingWithSingleBatch(int batchSize){
+    @ValueSource(ints = { 1000, 2000, 10000 })
```

Comment:
This was my thinking - Since { 1000, 2000, 10000 } appears three times in the file, if you change these values in some places and forget to change in others, it can create a problem. 

As you want (?) all three tests to run identical int values, I was thinking it would make sense to consolidate them to a method, even if you just return a list of 3 integers.

Reply:
I get what you are saying - we should err on the side of simplicity for the learners. I am fine leaving ValueSource as is.